lcm-cmake/lcmUtilities.cmake

```diff
@@ -260,6 +265,10 @@ function(lcm_wrap_types)
       if(_line MATCHES "^ *package +")
         # Get package name
         _lcm_extract_token(_package 1 "${_line}")
+        # Append package prefix if one was specified
+        if(_package_prefix)
```

Comment:
...another reason I'd prefer losing the superfluous variable (or at least adding `DEFINED` here) is that if someone uses `--package-prefix off`, they are going to be disappointed :wink:.